pkg/destroy/gcp/backendservice.go

```diff
@@ -1,9 +1,10 @@
 package gcp
 
 import (
+	"github.com/openshift/installer/pkg/types/gcp"
 	"github.com/pkg/errors"
 
-	compute "google.golang.org/api/compute/v1"
+	"google.golang.org/api/compute/v1"
 	"google.golang.org/api/googleapi"
```

Comment:
nit: we try to group imports like:
```
<stdlib>

<external>

<installer/openshift>
```
But that is not 100% consistent throughout the codebase.

Reply:
There's an automated tool to enforce this - do you want me to add a presubmit for the repo to enforce it?